### PR TITLE
feat order history customer

### PR DESCRIPTION
**Pull request by luthfiazhari**
### JIRA Link

[https://blocksphere2020.atlassian.net/browse/DBIO-325](url)

[https://blocksphere2020.atlassian.net/browse/DBIO-446](url)

### Changelog / Description

DBIO-325
Create Component for track Order history customer based on last status

DBIO-446
Create a parameter if the customer has staked the data, the stake data popup does not reappear

### CODE REVIEW

**Comment by luthfiazhari** [comment]
sorry if there are too many commits.
this is the commit for this pull request
(feat: Order history customer based on last status - 7cda938)

File: src/views/Dashboard/Customer/OrderHistoryDetail/index.vue [hunk 1]
```diff
@@ -91,32 +91,22 @@
...
-                    </v-col>
-                  </v-row>
-                </v-card-text>
-                <v-card-actions class="px-8 pb-5">
-                  <slot name="button"> </slot>
-                </v-card-actions>
-              </v-card>
-              <RatingBox>
-              </RatingBox>
+          <v-col cols="12" lg="6" md="6" xl="5" v-if="order.status == ORDER_FULFILLED || order.status == ORDER_PAID">
```
Comment (bukandicki) [inline src/views/Dashboard/Customer/OrderHistoryDetail/index.vue:94]: Mas @luthfiazhari ,, ini aku saranin buat di pindah ke computed aja mas,, kecualii dia di dalem loop

**Review by bukandicki — APPROVED** [review]
LGTM mas @luthfiazhari 👍🏻

File: src/views/Dashboard/Customer/OrderHistoryDetail/index.vue (hunk 1 above)
Comment (luthfiazhari) [reply src/views/Dashboard/Customer/OrderHistoryDetail/index.vue:94]: baik mas diki, terimakasih saran dan masukan nya.

**Review by kevinjanada — APPROVED** [review]

**Pull request merged**